node/common/asset_proof.ts

```diff
@@ -12,15 +12,6 @@ export class AssetProof {
   prevHash: Uint8Array = new Uint8Array();
   signature: Uint8Array = new Uint8Array();
 
-  /**
-   * @param {string} id
```

Comment:
It is redundant as it is, but shouldn't it describe what it is instead of deleting it as we do in the Javadoc?
https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns

Reply:
I'm not really sure about the TS coding style, but according to [Google one](https://google.github.io/styleguide/tsguide.html#redundant-comments), I guess it's not necessary to describe all properties if it's not ambiguous. (Do we need to describe them if it's a [parameter property](https://google.github.io/styleguide/tsguide.html#parameter-property-comments)?)

Reply:
@feeblefakie 
@jnmt 

Thank you for your advice.
I am sorry for the late reply.

The comment deleted in this PR was mainly for type hints in IDE. TypeScript can do this so I was thinking of deleting them.

But you also mention a point, it would be better if we can still use JSDoc to explain the parameters of the public classes (such as AssetProof, ContractExecutionResult, ... etc).

We didn't relly do this before. Let me complete this part in another PR.